taca/delivery/cli.py

```diff
@@ -0,0 +1,63 @@
+"""CLI for the delivery subcommand."""  # furure todo: rename to "deliver" when fully implemented and taca-ngi-pipieline is deprecated
+
+import click
+
+from taca.delivery import deliver
+
+
+@click.group()
+def delivery():
+    """Delivery methods entry point."""
+    pass
+
+
+# Stage data
+@delivery.command()
+@click.option(
+    "-f",
+    "--flowcells",
+    type=str,
+    help="Restrict action to specific flowcell(s). e.g.: 241122_VH00204_464_AAG77JJN5,241120_VH00202_453_AAG76JJM7",
+)
+@click.option(
+    "-s",
+    "--samples",
+    type=str,
+    help="Restrict action to specific sample(s). e.g.: P123_1001,P123_1002 (currently only Illumina and Aviti)",
+)
+@click.argument("project")
+def stage(project, flowcells, samples):
+    """Stage a project in DELIVERY and setup DDS delivery project."""
+    flowcells_to_stage = flowcells.split(",") if flowcells else None
+    samples_to_stage = samples.split(",") if samples else None
+    deliver.stage(project, flowcells=flowcells_to_stage, samples=samples_to_stage)
+
+
+# Upload data
+@delivery.command()
+@click.option(
+    "-d",
+    "--dds_id",
+    type=str,
+    required=True,
+    help="DDS PID generated by the stage command",
```

Comment:
Will the DDS PID still be generated by the stage command?

Reply:
No, you're right, I missed this. Thanks for spotting it 👍